### ISSUE TITLE
Can't use mouse to scroll

### ISSUE THREAD

**Issue by AgentCosmic** [akiyosi/goneovim#65]
When I use the the mouse to scroll, instead of scrolling, it just causes my undo history to pile up with nothing.

**Comment by akiyosi** [comment]
Thank you for the report.
Is the problem reproducible even with `Gonvim -u NONE`?

**Comment by AgentCosmic** [comment]
Works fine with the command. I did some testing and it seems like it has to do with something with these few lines:
```
let $HOME = expand('D:/Applications/Neovim/nvim')
set runtimepath+=$HOME
runtime mswin.vim

```

**Comment by akiyosi** [comment]
Sorry, I'm late.

I looked into this and I detect the cause.
This is a probrem of key mapping for MS-Windows shortcut.
https://github.com/neovim/neovim/blob/master/runtime/mswin.vim#L76-L78

Gonvim mouse scroll feature uses the same mapping `<C-Y>`, which causes the probrem.
Currently, the only workaround is to comment out the corresponding line in mswin.vim.

**Comment by AgentCosmic** [comment]
Hi,
Unfortunately I use a plugin "mattn/emmet-vim" which maps to <c-y> automatically so I can't fix it without disabling emmet. Will this be fixed?

**Comment by akiyosi** [comment]
Ok,
I will plan to fix this in the next version v0.4.0.
